config/nightly_configs/amip_coarse.yml

```diff
@@ -0,0 +1,28 @@
+FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
+anim: false
+atmos_config_file: "config/longrun_configs/amip_target_diagedmf.yml"
+dt: "240secs"
+dt_cpl: 240
+dt_save_state_to_disk: "30days"
+dt_save_to_sol: "30days"
+energy_check: false
+h_elem: 8
+hourly_checkpoint: true
+hourly_checkpoint_dt: 720
+land_albedo_type: "map_temporal"
+mode_name: "amip"
+mono_surface: false
+netcdf_output_at_levels: true
+output_default_diagnostics: true
+rayleigh_sponge: true
+start_date: "20100101"
+surface_setup: "PrescribedSurface"
+t_end: "1095days"
```

Comment:
Having t_end exactly the same as three years may result in missing final month diagnostics due to some floating point errors in time. I used 1098 days to be safe with leap year and floating point errors (although to be accurate, 1097 days should be good enough).